app/lib/services/auth_service.rb

```diff
@@ -19,7 +19,7 @@ def authorize!(identity, cert_issue_req)
 
     def decode(token)
       # Decode a JWT access token using the configured base.
-      body = JWT.decode(token, Rails.application.config.astral[:jwt_signing_key])[0]
+      body = JWT.decode(token, Rails.configuration.astral[:jwt_signing_key])[0]
```

Comment:
You're using .first elsewhere and [0] here, should we standardize our style?

Reply:
Interesting question. I'd say `.first` makes sense with a list of similar things (AR records, eg) but `JWT.decode` is producing an ordered data structure where the elements are different (only element 0 works). So it has a slightly different flavor?

Reply:
Just doesn't seem as delightfully Rubyish, but your call.